src/sentry/sentry_apps/api/serializers/app_platform_event.py

```diff
@@ -1,49 +1,93 @@
+from collections.abc import Mapping
 from time import time
+from typing import Any, Generic, TypedDict, TypeVar
 from uuid import uuid4
 
+from sentry.sentry_apps.models.sentry_app import SentryAppActionType, SentryAppResourceType
+from sentry.sentry_apps.models.sentry_app_installation import SentryAppInstallation
+from sentry.sentry_apps.services.app.model import RpcSentryAppInstallation
+from sentry.users.models.user import User
+from sentry.users.services.user import RpcUser
 from sentry.utils import json
 
 
-class AppPlatformEvent:
+class AppPlatformEventActor(TypedDict):
+    type: str
+    id: str | int
+    name: str
+
+
+class AppPlatformEventInstallation(TypedDict):
+    uuid: str
+
+
+T = TypeVar("T", bound=Mapping[str, Any])
```

Comment:
Can we restrict the values any further? `Any` is something I'd like to move away from whenever possible.